### ISSUE TITLE
Crash too long filename

### ISSUE THREAD

**Issue by imtrobin** [ashe23/ProjectCleaner#32]
Got a crash when I try to open Project Cleaner.

```
Assertion failed: false [File:D:/Build/++UE4/Sync/Engine/Source/Runtime/Core/Private/UObject/UnrealNames.cpp] [Line: 1830] FName's 1023 max length exceeded. Got 3344 characters excluding null-terminator:..xxxxxx...),ImageStretch=ScaleToFit,Padding=(Left=0.000000,Top=0.000000,Right=0.000000,Bottom=0.000

UE4Editor_ProjectCleaner!FProjectCleanerDataManager::FindIndirectAssets() [D:\build\++Portal\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\ProjectCleaner\Source\ProjectCleaner\Private\Core\ProjectCleanerDataManager.cpp:577]
UE4Editor_ProjectCleaner!FProjectCleanerDataManager::AnalyzeProject() [D:\build\++Portal\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\ProjectCleaner\Source\ProjectCleaner\Private\Core\ProjectCleanerDataManager.cpp:56]
UE4Editor_ProjectCleaner!FProjectCleanerManager::Update() [D:\build\++Portal\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\ProjectCleaner\Source\ProjectCleaner\Private\Core\ProjectCleanerManager.cpp:37]
UE4Editor_ProjectCleaner!FProjectCleanerModule::PluginButtonClicked() [D:\build\++Portal\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\ProjectCleaner\Source\ProjectCleaner\Private\ProjectCleaner.cpp:80]
```

**Comment by ashe23** [comment]
Hi @imtrobin 
Thanks for report.
Can you tell which engine version you use? and did you update plugin to 1.6.0?
Because it very similar to issue that was fixed by this #25 .

**Comment by imtrobin** [comment]
Yes, 1.6.0 fixed it. Sorry for false alarm.
